client/src/main/java/org/spine3/base/Stringifiers.java

```diff
@@ -90,4 +91,42 @@ private Stringifiers() {
                                     .convert(str);
         return result;
     }
+
+    /**
+     * Obtains {@code Stringifier} for map.
+     *
+     * @param keyClass   the class of keys are maintained by this map
+     * @param valueClass the class  of mapped values
+     * @param <K>        the type of keys are maintained by this map
+     * @param <V>        the type of mapped values
+     * @return the stringifier for the map
+     */
+    public static <K, V> Stringifier<Map<K, V>> mapStringifier(Class<K> keyClass,
+                                                               Class<V> valueClass) {
+        checkNotNull(keyClass);
+        checkNotNull(valueClass);
+        final Stringifier<Map<K, V>> mapStringifier = new MapStringifier<>(keyClass, valueClass);
+        return mapStringifier;
+    }
+
+    /**
+     * Obtains {@code Stringifier} for map.
```

Comment:
Please make the documentation different from the method above, not just the param list.